frameit/frames_generator/Rakefile

```diff
@@ -98,13 +125,20 @@ task(:generate_device_frames) do
   puts("Push the changes, and run `fastlane frameit update_frames` to ensure everything worked as expected".green)
 end
 
+def should_skip_device?(device_file_name)
+  DEVICES_TO_SKIP.each do |device_to_skip|
+    return true if device_file_name.start_with?(device_to_skip)
+  end
+  false
+end
+
 def confirm
   puts("Press any key when you're ready to continue".yellow)
   STDIN.gets
 end
 
 def sanitize_filename(filename)
-  filename.gsub('Grey', 'Gray')
+  filename.gsub('Grey', 'Gray').gsub(' - ', ' ')
```

Comment:
What is the ` - ` replacement used for?

Reply:
This is because the naming of the devices from Facebook is inconsistent and the Google Pixel devices contain a dash between their name and color, e. g. "Google Pixel 3 XL - Clearly White.png". We don't want to have that dash in the device name.

Reply:
Ok, best add that as a comment (probably same for the Grey/Gray thing).

Reply:
OK, added a comment in the new commit.